tools/Common/IceRpc.BuildTelemetry/TelemetryTask.cs

```diff
@@ -23,9 +23,24 @@ public class TelemetryTask : ToolTask
     public string Idl { get; set; } = "";
 
     /// <summary>
-    /// Gets or sets the if the compilation contained any Slice1 files.
+    /// Gets or sets a value indicating whether the compilation contained any Slice1 files.
     /// </summary>
-    public string? ContainsSlice1 { get; set; } = "";
+    public bool ContainsSlice1 { get; set; } = false;
```

Comment:
You don't need to and should not set a property to its default value. I am surprised you didn't get a warning. Same for the next property.